src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -3682,4 +3781,13 @@ Activate by holding the key for the character you want to add an accent to, then
     <value>Administrator: PowerToys Settings</value>
     <comment>Title of the settings window when running as administrator</comment>
   </data>
+  <data name="DashboardTitle.Text" xml:space="preserve">
+    <value>Dashboard</value>
+  </data>
+  <data name="Shell_Dashboard.Content" xml:space="preserve">
+    <value>Dashboard</value>
+  </data>
+  <data name="GPO_IsSettingForcedText.Text" xml:space="preserve">
```

Comment:
Equals `.Title` now?

Reply:
This resource is used for the <TextBlock x:Uid="GPO_IsSettingForcedText" ...> to set its Text. I'm not sure about the .Title you mentioned. Please let me know if you see a problem with it. Thanks a lot for the review!

Reply:
No problem. Just saw the same text/content. (at line 3498)